File: 01.ProgrammingFundamentals/Java/02.ConditionalStatementsAndLoops/Lab/src/P05_ForeignLanguages.java
import java.util.Scanner;
public class P05_ForeignLanguages {
    public static void main(String[] args) {

        Scanner console = new Scanner(System.in);
        String country = console.nextLine().toLowerCase();

        switch (country) {
            case "usa":
            case "england":
                System.out.println("English");
                break;
            case "spain":
            case "argentina":
            case "mexico":
                System.out.println("Spanish");
                break;
            default:
                System.out.println("unknown");
                break;
        }
    }
}

File: 01.ProgrammingFundamentals/C#/02.ConditionalStatementsAndLoops/Lab/P05_ForeignLanguages/P05_ForeignLanguages.cs
using System;

namespace P05_ForeignLanguages
{
    class P05_ForeignLanguages
    {
        static void Main(string[] args)
        {
            string country = Console.ReadLine();
            string language;
            switch (country)
            {
                case "USA":
                case "England":
                    language = "English";           
                    break;
                case "Spain":
                case "Argentina":
                case "Mexico":
                    language = "Spanish";
                    break;
                default:
                    language = "unknown";
                    break;
            }
            Console.WriteLine(language);
        }
    }
}
